doc/md/getting-started/02-inspecting.md

```diff
@@ -161,3 +161,7 @@ read the [SQL Syntax](../ddl/sql.md) documentation.
 
 In the next section, we will see how we can modify our database's schema by 
 applying a modified Atlas HCL file using the Atlas CLI.
+
+## SSL Mode
```

Comment:
The getting started tutorial uses MySQL for the demo. I think it's best to add a new page in the CLI section that documents DSNs and explains a bit about them and add this in a section about postgres. 

Then we can put a [Docusaurus Admonition](https://docusaurus.io/docs/next/markdown-features/admonitions) in this page that references there (e.g. "If you're using PostgreSQL, here's documentation about building the connection string" or something). 

Can be done in a separate PR.

Reply:
This makes sense to me. 

Could you clarify where you imagine this living? I am interpreting your instructions as
1. `git mv doc/md/cli.md doc/md/cli/reference.md`
2. `touch doc/md/cli/dsn.md`
3. Move this blurb into `doc/md/cli/dsn.md` and add some details about DSNs in more detail

before I make changes want to make sure I'm on track

Reply:
sounds good, notice that doc/md/cli.md is a generated file that is generated by [this script](https://github.com/ariga/atlas/blob/master/internal/typedoc/main.go) so when you move that be sure to update that. 

I added instructions run to test the doc site locally [here](https://atlasgo.io/contributing#contributing-documentation) , that would be the best way to see that everything looks good before committing